### DISCUSSION TITLE
also having the housing we have a problem

### DISCUSSION THREAD

**Discussion by yowshi** [TweakScale/TweakScale#272] [Support] [answered]
I have tried reading the posts on this, including the one by you elsewhere to see if I could figure it out. sadly no luck any help would be appreciated.
[KSP.log](https://github.com/net-lisias-ksp/TweakScale/files/10017780/KSP.log)

**Comment by Lisias** [comment]
@yowshi , 

Damn… O made a mistake on the deployment, as it appears…

I'm checking it. Completely remove 2.4.6.17r2 and install back 2.4.6.16 for while!

**Comment by Lisias** [comment]
@yowshi 

I just downloaded the SpaceDock package, and it's identical to the respective Github one. And both worked fine on my 1.12.4 test bed!

```
[LOG 05:16:52.016] [KSPe.Light.TweakScale] Version 2.4.2.4 /L for TweakScale /L
[LOG 05:16:52.133] [TweakScale] Version 2.4.6.17 /L
[LOG 05:16:54.662] [KSPe.Binder] Hooked.
[ERR 05:16:54.666] ADDON BINDER: Cannot resolve assembly: Scale.PartDB.19x

[LOG 05:16:54.668] [KSPe.Binder] Looking for Scale.PartDB.19x.dll on GameData/TweakScale/Plugins/PluginData/...
[LOG 05:16:54.668] [KSPe.Binder] Found it on GameData/TweakScale/Plugins/PluginData/Scale.PartDB.19x.dll.
[LOG 05:16:54.674] [TweakScale] Support for KSP 1.9.0 to 1.12.4 Version 2.4.6.17 /L
```

**Comment by Lisias** [comment]
The problem is only happening on Windows!!!! DAMN!

**Reply by Lisias** [reply]
Full Disclosure for my future self: the MacOS testbed was contaminated, and it masked the problem.

Windows's test bed reproduced the problem because the test bed was shinning new.

A new test bed for MacOS was built for 1.12.4.

**Comment by Lisias** [comment, marked as answer]
Found the problem. I don't have the slighest idea why it affected only Windows users (it should had been screwing up MacOS too, at least).

Fixing it. A new release will be published in the next hour at worst (assuming a meteor don't hit me first).

**Reply by yowshi** [reply]
@Lisias 
wow and i thought it was just some other mod, thanks for fixing this so quickly.

**Reply by Lisias** [reply]
Usually it's some other mod. But this time was me… :P 

Oh, well. Everybody borks - Gregory Kerman, M.D.
